specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2024-03-01-preview/Capabilities.json

```diff
@@ -271,6 +271,7 @@
       "type": "string",
       "description": "The version of a server.",
       "enum": [
+        "16",
```

Comment:
check if this is required even for ServerCapabilities API

Reply:
Validated and updated examples